examples/subscribing_to_stream.rs

```diff
@@ -0,0 +1,112 @@
+use eventstore::{
+    Client, Credentials, EventData, ExpectedVersion, Origin, Position, ReadResult, SubEvent,
+    SubscribeToAllOptions, SubscribeToStreamOptions, SubscriptionFilter,
+};
+use futures::TryStreamExt;
+use serde::{Deserialize, Serialize};
+use std::error::Error;
+use uuid::Uuid;
+
+type Result<A> = std::result::Result<A, Box<dyn Error>>;
+
+pub async fn subscribe_to_stream(client: &Client) -> Result<()> {
+    // region subscribe-to-stream
+    client
+        .subscribe_to_stream("some-stream", &Default::default())
```

Comment:
**FIX:** This snippet is missing the subscription logic (callback/handler) as it is in other samples.

Reply:
You have it at line 45. Do you want me to move it to that snippet instead?

Reply:
It'd be good to show that here (so preferably copy than move) to be consistent with other samples and show the users the happy path in the first sample they see.